File: Csharp/Models/TextContent.cs
namespace Models
{
    public class TextContent : Content
    {
        public string Body { get; set; }

        public TextContent() 
        {
            Type = "text"; 
        }

        public TextContent(string body)
        {
            Body = body;
            Type = "text"; 
        }
    }
}

File: Java/src/main/java/com/backend/java/model/TextContent.java
package com.backend.java.model;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonTypeName;
import jakarta.persistence.Entity;
import jakarta.persistence.Table;

@Entity
@Table(name = "text_content")
@JsonTypeName("text")
public class TextContent extends Content {
    @JsonProperty("body")
    private String body;

    // Constructors, getters, and setters
    public TextContent() {}

    public TextContent(String body) {
        this.body = body;
    }

    public String getBody() {
        return body;
    }

    public void setBody(String body) {
        this.body = body;
    }

    @Override
    public String toString() {
        return "TextContent{" +
                "body='" + body + '\'' +
                '}';
    }
}
